### ISSUE TITLE
VS 2019 Preview 1 is reporting members as unused when they are

### ISSUE THREAD

**Issue by CoolDadTx** [dotnet/roslyn#32702]
**Version Used**:  VS2019 Preview 1

**Steps to Reproduce**:

Note that this is hit or miss. It randomly seems fine with some methods but not others.

```csharp
static partial class ContainerConfig
{
   static partial void Configure( ContainerBuilder builder, Assembly assembly )
    {
     builder.ConfigureMvcBindings()
                .ConfigureCoreBindings();
     builder.ConfigureLogging();
   }  

   //CodeLens says ref count = 1, method is marked as unused
   private static ContainerBuilder ConfigureCoreBindings( this ContainerBuilder builder )
   {
       return builder;
   }

   //CodeLens says ref count = 1, method is seen as used
   public static ContainerBuilder ConfigureMvcBindings( this ContainerBuilder builder )
   {
      return builder;
   }

   //CodeLens says ref count = 1, method is seen as unused
   private static ContainerBuilder ConfigureLogging( this ContainerBuilder builder )
   {
      return builder;
   }
}
```

This seems to be working correctly in VS 2018 15.9.4. Closing and reopening VS doesn't change anything.

**Expected Behavior**:

Members that are being used are not marked as unused.

**Actual Behavior**:

Some members that are clearly being used in the same source file are still being marked as unused. The CodeLens reference count is correct but Roslyn doesn't seem to detect the reference.

![unusedmembers](https://user-images.githubusercontent.com/3778674/51635061-80faaa00-1f1b-11e9-94b5-0e0418b92338.PNG)

**Comment by mavasani** [comment]
@CoolDadTx Thanks for reporting the issue. Couple of things:
1. Our unused members analyzer currently only works on private members, hence you never see them flagging non-private members.
2. I was able to repro your issue - seems related to partial methods not getting analyzed when partial method definitions are defined across multiple files.
